src/FileDownloader/ChunkFileDownloader.cs

```diff
@@ -168,7 +168,6 @@ private async Task DequeueAndDownloadChunksAsync(CancellationToken ct = default)
                 {
                     if (workerFailedDownloadingOnce)
                     {
-                        await Task.Delay(1000, ct);
```

Comment:
This is here to make the server allow connections from the client again. Otherwise we will instantly get rate-limited.